heron/common/src/python/argparser.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
I am not sure if you can call it `argparser.py` since it only contains `HeronRCArgumentParser`. Should we rename it to, for example, `rcargparser.py`?